graphql/resolvers.go

```diff
@@ -594,6 +594,23 @@ func (r *mutationResolver) SetTaskPriority(ctx context.Context, taskID string, p
 	return &apiTask, nil
 }
 
+func (r *mutationResolver) SchedulePatch(ctx context.Context, patchID string, reconfigure PatchReconfigure) (*restModel.APIPatch, error) {
+	patchUpdateReq := PatchVariantsTasksRequest{}
+	patchUpdateReq.BuildFromGqlInput(reconfigure)
+
+	// do not need to check error after fetching version bc version can be nil
+	version, _ := r.sc.FindVersionById(patchID)
```

Comment:
You should still error check here in case there's a database error

Reply:
there is no distinction between a db error and a nil patch error

Reply:
used `strings.Contains` to determine if error is gimlet 404